core/src/cubos/core/gl/ogl_render_device.cpp

```diff
@@ -1232,8 +1232,8 @@ OGLRenderDevice::OGLRenderDevice()
 {
     // Set the debug message callback
     // TODO: disable this on release for performance reasons (?)
-    glEnable(GL_DEBUG_OUTPUT);
-    glDebugMessageCallback(messageCallback, nullptr);
+    // glEnable(GL_DEBUG_OUTPUT);
+    // glDebugMessageCallback(messageCallback, nullptr);
 
```

Comment:
Don't just disable this - instead, [fix this](https://github.com/GameDevTecnico/cubos/issues/740). You can do it in this PR too, it is enough to get the OpenGL version and only call these funcs if it is high enough (check OGLRenderDevice::getProperty for an example).